configurations/manager/backup-1TB-gce.yaml

```diff
@@ -0,0 +1,40 @@
+test_duration: 720
```

Comment:
Why is it under configurations and not test-cases/manager/?

Reply:
My goal was to build it ontop of the 1TB longevity, and seem like it would be feasible